### ISSUE TITLE
Convolutional Neural Network for Brain Lesion Segmentation in MRI Images

### ISSUE THREAD

**Issue by 673958639** [673958639/Convolutional-Neural-Network-for-Brain-Lesion-Segmentation-in-MRI-Images#1]
Among brain lesions, brain tumors are aggressive and dangerous and may lead to shortened patient life span. Therefore, effective treatment modalities and observation of the pathology are necessary. Magnetic resonance imaging (MRI) is an imaging technique used to evaluate tumors, but MRI generates a huge amount of data and has limited segmentation efficiency and accuracy. In this work, four popular deep convolutional neural networks (U-NET, DeepLab, FCN and SegNet) for image segmentation are constructed and compared. This comparison reveals the tradeoff between achieving effective segmentation and segmentation accuracy. Using deep learning, specifically convolutional neural network methods, to build and train models, brain lesions can be identified from MRI images. After the model was able to identify brain injury, the deep learning model was improved by adjusting the model structure, implementing data enhancement, and searching for optimal hyperparameters. We also elaborate on the implementation details and evaluation criteria to improve its segmentation benchmark and performance based on the original model. 
https://github.com/673958639/Convolutional-Neural-Network-for-Brain-Lesion-Segmentation-in-MRI-Images
